### PR TITLE
i18n: add Translations fr more content

### PR DESCRIPTION
**Pull request by VanSlanzar**
- I get this, while the regex have been added for `Relative Virtue`.

```
$ npm run find-translations -- --locale=fr -f eureka_hydatos

> cactbot@0.24.19 find-translations C:\Users\gamer\Documents\GitHub\cactbot
> npm run util -- findTranslations "--locale=fr" "-f" "eureka_hydatos"


> cactbot@0.24.19 util C:\Users\gamer\Documents\GitHub\cactbot
> node --loader ts-node/esm util/index.ts "findTranslations" "--locale=fr" "-f" "eureka_hydatos"

(node:27616) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
C:\Users\gamer\Documents\GitHub\cactbot\ui\raidboss\data\04-sb\eureka\eureka_hydatos.ts:102 BA Seal missing timelineReplace for regex '^^(?:00)\|(?:[^|]*)\|(?:0839)\|(?:[^|]*)\|(?:.*will be sealed off.*?)\|' (but have netRegexFr)
C:\Users\gamer\Documents\GitHub\cactbot\ui\raidboss\data\04-sb\eureka\eureka_hydatos.ts:644 BA AV Eidos Relative Virtue Astral incorrect timelineReplace for regex, got '^^(?:00)\|(?:[^|]*)\|(?:[^|]*)\|(?:[^|]*)\|(?:vertu relative gains the effect of astral essence.*?)\|', needed netRegexFr '^^(?:00)\|(?:[^|]*)\|(?:[^|]*)\|(?:[^|]*)\|(?:vertu relative bénéficie de l'effet.*?bras de lumière.*?)\|'
C:\Users\gamer\Documents\GitHub\cactbot\ui\raidboss\data\04-sb\eureka\eureka_hydatos.ts:659 BA AV Eidos Relative Virtue Umbral incorrect timelineReplace for regex, got '^^(?:00)\|(?:[^|]*)\|(?:[^|]*)\|(?:[^|]*)\|(?:vertu relative gains the effect of umbral essence.*?)\|', needed netRegexFr '^^(?:00)\|(?:[^|]*)\|(?:[^|]*)\|(?:[^|]*)\|(?:vertu relative bénéficie de l'effet.*?bras de ténèbres.*?)\|'

```
- if I add it  `will be sealed off`, I get a collision. Can you help me with these 3 problems.
```
 AssertionError: ui/raidboss/data/04-sb/eureka/eureka_hydatos.ts:locale fr: pre-translation collision on replaceSync ' 00:0839::The Shin-Zantetsuken Containment Unit will be sealed off' for 'will be sealed off' and '(?<=00:0839::|00\|[^|]*\|0839\|\|)(.*) will be sealed off(?: in (?:[0-9]+ seconds)?)?': expected ' 00:0839::Fermeture d(?:e|u|es) (?:l\'|la |les? )?The Shin-Zantetsuken Containment Unit dans' to equal ' 00:0839::The Shin-Zantetsuken Containment Unit Fermeture d(?:e|u|es) .*? dans'
      + expected - actual

      - 00:0839::Fermeture d(?:e|u|es) (?:l'|la |les? )?The Shin-Zantetsuken Containment Unit dans
      + 00:0839::The Shin-Zantetsuken Containment Unit Fermeture d(?:e|u|es) .*? dans

```

### CODE REVIEW

File: ui/eureka/eureka_config.ts [hunk 1]
```diff
@@ -44,8 +44,7 @@ UserConfig.registerOptions('eureka', {
       name: {
         en: 'Prefer complete names for Skirmishes/Critical Engagements',
         de: 'Bevorzuge komplette Namen für Scharmützel/Kritische Gefechte',
-        fr:
-          'Préférer les noms complet pour les escarmouches/Affrontements Cruciaux dans Bozja/Zadnor',
+        fr: 'Préférer les noms complet pour les Escarmouches/Affrontements Cruciaux dans Bozja/Zadnor',
```
Comment (VanSlanzar) [inline ui/eureka/eureka_config.ts:47]: @quisquous, is the line break required?

File: ui/raidboss/emulator/translations.ts [hunk 2]
```diff
@@ -149,8 +149,7 @@ const emulatorDeleteModal: Translation = {
   '.deleteDBModal .modal-body': {
     en: '<p>You are about to delete the encounter database. Are you sure?</p>',
     de: '<p>Du bist kurz davor die Kampfdatenbank zu löschen. Bist du sicher?</p>',
-    fr:
-      '<p>Vous êtes sur le point de supprimer la base de données du combat. En êtes-vous sûr ?</p>',
+    fr: '<p>Vous êtes sur le point de supprimer la base de données du combat. En êtes-vous sûr ?</p>',
```
Comment (VanSlanzar) [inline ui/raidboss/emulator/translations.ts:152]: Same ?

File: ui/eureka/eureka_config.ts (hunk 1 above)
Comment (quisquous) [reply ui/eureka/eureka_config.ts:47]: Yeah, if the line is longer than the line length (100 characters) then the line break is required.  That's why the lint step is complaining.  You could also shorten it and remove the Bozja/Zadnor part too.

Comment (VanSlanzar) [reply ui/eureka/eureka_config.ts]: Thank You.

**Comment by quisquous** [comment]
```diff
diff --git a/ui/raidboss/data/04-sb/eureka/eureka_hydatos.ts b/ui/raidboss/data/
04-sb/eureka/eureka_hydatos.ts
index ecdca8026..833b946e2 100644
--- a/ui/raidboss/data/04-sb/eureka/eureka_hydatos.ts
+++ b/ui/raidboss/data/04-sb/eureka/eureka_hydatos.ts
@@ -1148,7 +1148,7 @@ const triggerSet: TriggerSet<Data> = {
     {
       'locale': 'fr',
       'replaceSync': {
-        '7 minutes have elapsed since your last activity..*?': 'Votre personnag
e est inactif depuis 7 minutes.*?',
+        '7 minutes have elapsed since your last activity.': 'Votre personnage e
st inactif depuis 7 minutes',
         'Absolute Virtue': 'Vertu absolue',
         'Arsenal Centaur': 'Centaure de l\'Arsenal',
         'Art': 'Art',
@@ -1161,7 +1161,7 @@ const triggerSet: TriggerSet<Data> = {
         'Ozmashade': 'Ombre d\'Ozma',
         'Proto Ozma(?! containment)': 'Proto-Ozma',
         'Raiden': 'Raiden',
-        'Relative Virtue': 'Vertu relative',
+        'Relative Virtue(?! gains)': 'Vertu relative',
         'Shadow': 'Ombre de Proto-Ozma',
         'Streak Lightning': 'Éclair chargeant',
         'The Lance of Virtue Containment Unit': 'l\'enceinte de confinement de
la lance de la vertu',
```

These solve some of the issues above.  I think the "will be sealed" issue is a bug with the test.  Feel free to file an issue and I'll try to take a look at it, but in the meantime I don't think you can fix it by just changing translations.

**Pull request merged**